dockerfiles/tasks.py

```diff
@@ -138,21 +138,12 @@ def restart(c, containers):
             c.run(f'{DOCKER_COMPOSE_COMMAND} restart nginx', pty=True)
             break
 
-@task(help={
-    'only_required': 'Only pull the required image (used by default for clonning). Use it if you don\'t need all images (default: False)',
-})
-def pull(c, only_required=False):
+@task()
+def pull(c):
     """Pull all docker images required for build servers."""
     images = [
-        ('ubuntu-20.04-2022.02.16', 'ubuntu-20.04'),
+        ('ubuntu-22.04-2023.03.09', 'ubuntu-20.04-2022.02.16'),
```

Comment:
Actually, this is wrong 🙃 -- the second argument should be the tag... I will fix this.

Thanks for commenting here because that made me realize there is a mistake.

Reply:
Yea, would be good to also have that be a settings, instead of hard coded :)